harness/engine/profiles.go

```diff
@@ -23,6 +23,9 @@ var profileList Profiles
 
 // Profile1 builds the default dusk.toml definition
 func Profile1(index int, node *DuskNode, walletPath string) {
+
```

Comment:
To keep the code style consistent, don't put whitespace between the function declaration and body

Reply:
Maybe to ensure this and any other Style rules, `go fmt` and  `go linter` should be extended.

Reply:
There's an issue open to create a custom GitHub action which also ensures new files have the proper license header. This will be a good addition in that case. I will extrapolate it into an issue, but please do delete this line for the PR.

Reply:
Done